how-to/customize-workspace/client/src/platform-storage-shapes.ts

```diff
@@ -0,0 +1,34 @@
+export interface IPlatformStorage {
```

Comment:
Couldnt the whole definition be generic, instead of passing to each method i.e. `IPlatform<T>` and remove `<T`> from the methods

Reply:
done